addons/l10n_sk/i18n/l10n_sk.pot

```diff
@@ -0,0 +1,120 @@
+# Translation of Odoo Server.
```

Comment:
Hmm, weird. Seems like we didn't have any translations for Slovakian yet 🤔 
Well, at least we have some now 😅 Thanks!